upgraderesponder/service.go

```diff
@@ -22,6 +22,9 @@ const (
 	VersionTagLatest  = "latest"
 	AppMinimalVersion = "v0.0.1"
 
+	InfluxDBContinuousQueryFmt = "cq_by_%s_sampling"
```

Comment:
Should we change to `"cq_by_%s_down_sampling"` because we are doing down sampling? It matches other queries names